### PR TITLE
Fix TCK failure for EJB32 tests

### PR DESCRIPTION
**Pull request by arjantijms**
Use EclipseLink 4.0.3 instead of 4.0.4
Weld back to 5.1.3

### CODE REVIEW

File: appserver/tests/tck/platform-tck-runner/src/main/java/org/glassfish/main/tests/tck/ant/TckRunner.java [hunk 1]
```diff
@@ -183,7 +183,7 @@ private void deleteDirectory(final Path directory) throws IOException {
     }
 
     private Process startBash(final String command) throws IOException {
-        ProcessBuilder bash = new ProcessBuilder("/bin/bash", "-c", command).inheritIO().directory(cfg.getTargetDir());
+        ProcessBuilder bash = new ProcessBuilder("/bin/zsh", "-c", command).inheritIO().directory(cfg.getTargetDir());
```
Comment (dmatej) [inline appserver/tests/tck/platform-tck-runner/src/main/java/org/glassfish/main/tests/tck/ant/TckRunner.java:186]: I don't have zsh and i don;t see it as some standard ;-)

**Review by dmatej — CHANGES_REQUESTED** [review]

File: appserver/tests/tck/platform-tck-runner/src/main/java/org/glassfish/main/tests/tck/ant/TckRunner.java (hunk 1 above)
Comment (arjantijms) [reply appserver/tests/tck/platform-tck-runner/src/main/java/org/glassfish/main/tests/tck/ant/TckRunner.java:186]: sorry, accidentally in the commit

**Comment by dmatej** [comment]
See https://github.com/eclipse-ee4j/eclipselink/issues/2219 - seems this is the problem.

**Review by dmatej — APPROVED** [review]

**Comment by dmatej** [comment]
... and this PR very probably broke it: https://github.com/eclipse-ee4j/eclipselink/pull/2181

**Pull request merged**